bundle/nxiq-operator-certified.clusterserviceversion.yaml

```diff
@@ -122,15 +151,15 @@ spec:
     | `iq.adminPort`       | Port of the application connector. Must match the value in the `configYaml` property | `8071`            |
     | `iq.memory`          | The amount of RAM to allocate                                | `1Gi`             |
     | `iq.licenseSecret`   | The base-64 encoded license file to be installed at startup  | `""`              |
-    | `iq.configYaml`      | A YAML block which will be used as a configuration block for IQ Server. | See `values.yaml` |
+    | `iq.configYaml`      | A YAML block which will be used as a configuration block for IQ Server. | See YAML. |
```

Comment:
What is `YAML` in this context?

Reply:
It's the example custom resource yaml that it generates and presents upon clicking "create instance" that looks a lot like the values.yaml. I almost want to say nothing in that place, since I have nothing to offer over what's generated, but I guess I can say something better. Maybe "See custom resource when creating instance."